### PR TITLE
#2905 Issuer registration flow UX fixes

### PR DESCRIPTION
**Pull request by desfero**

### CODE REVIEW

File: app/components/wallet-selector/ledger/WalletLedgerInitComponent.tsx [hunk 1]
```diff
@@ -135,8 +139,4 @@ export const WalletLedgerInit = compose<IWalletLedgerInitComponentProps, {}>(
       errorMessage: state.ledgerWizardState.errorMsg,
     }),
   }),
-  branch<IWalletLedgerInitComponentProps>(
```
Comment (Moejoe90) [inline app/components/wallet-selector/ledger/WalletLedgerInitComponent.tsx]: why was this removed? 
Couldn't we user `withContainer` for ` <LedgerHeader />`

File: app/utils/withContainer.unsafe.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import * as React from "react";
 import { ComponentEnhancer, nest } from "recompose";
 
-export function withContainer(Layout: React.ComponentType<any>): ComponentEnhancer<any, any> {
```
Comment (Moejoe90) [inline app/utils/withContainer.unsafe.tsx]: :+1:

File: app/components/wallet-selector/WalletSelector.tsx [hunk 3]
```diff
@@ -41,128 +45,141 @@ interface IDispatchProps {
...
-                      <FormattedMessage id="wallet-selector.tabs.neuwallet-login" />
-                    ) : (
-                      <FormattedMessage id="wallet-selector.tabs.neuwallet-register" />
-                    )}
-                  </ButtonLink>
+}) => {
+  const isIssuerWithOnlyLedgerAllowed =
+    userType === "issuer" && process.env.NF_ISSUERS_CAN_LOGIN_WITH_ANY_WALLET !== "1";
+
+  return (
```
[the remark this answers is not in the document]
Comment (desfero) [reply app/components/wallet-selector/WalletSelector.tsx]: yeah, that makes sense
